### ISSUE TITLE
Certification Upon Completion of a Course

### ISSUE THREAD

**Issue by xdvrx1** [githubtraining/introduction-to-github#53]
## Feature Request

Hi there. This is not my first time to use GitHub. I am an active contributor. But I just
tried this for the sake of a certificate. I expected it but there was none.
This will be more encouraging for others to take if you will be issuing 
a certificate for every course completion. 

**Describe your ideal solution**
So, I highly suggest you issue a certificate for every course completion. They need that
for their credentials. Other companies are doing that and they are in demand now.

**Comment by investigative** [comment]
Got it. Thx for the heads up!

**Comment by xdvrx1** [comment]
thanks for the response,

**Comment by hectorsector** [comment]
Thanks for the suggestion, @xdvrx1! I'm working on this idea and would to chat with you further if you're willing. If so, please reach out at learninglab@github.com so we can coordinate the best day/time?

**Comment by jedpepper** [comment]
I am a beginner here so I don't really know what I am exactly doing 👍

**Comment by xdvrx1** [comment]
@jedpepper, follow me here in GitHub, I'll follow you back, and I have several tutorials for beginners <https://github.com/xdvrx1>, thanks

**Comment by Dhanush560** [comment]
> I am a beginner here so I don't really know what I am exactly doing 👍

Same problem bro

**Comment by RJHP** [comment]
I totally agree ... Github requires certification in every course ... isn't a certificate our mark of passing?
